doc/association-rules.rst

```diff
@@ -0,0 +1,74 @@
+Association Rules
+=================
+
+.. figure:: icons/association-rules.png
+
+Induction of association rules.
+
+Signals
+-------
+
+**Inputs**:
+
+-  **Data**
+
+Data set
```

Comment:
This is a template from the rest of Orange docs. If you want changes, that would require changing the entire Orange documentation.